src/scripts/multiDownload.js

```diff
@@ -38,24 +38,26 @@ function download(url) {
     const a = document.createElement('a');
     a.download = '';
     a.href = url;
-    // firefox doesn't support `a.click()`...
-    a.dispatchEvent(new MouseEvent('click'));
+    a.click();
 }
 
 export default function (urls) {
     if (!urls) {
         throw new Error('`urls` required');
     }
 
-    if (typeof document.createElement('a').download === 'undefined') {
+    if (typeof document.createElement('a').download === 'undefined'
+        || browser.iOS) {
         return fallback(urls);
     }
 
     let delay = 0;
 
     urls.forEach(function (url) {
         // the download init has to be sequential for firefox if the urls are not on the same domain
-        if (browser.firefox && !sameDomain(url)) {
+        // desktop safari also requires the delay
+        if ((browser.firefox && !sameDomain(url))
```

Comment:
I'd like to add this delay to all browsers. On browsers that seem to be happy with this will still missing some episodes without the delay.